### ISSUE TITLE
ledger_device_sdk rust doc generate failed

### ISSUE THREAD

**Issue by Pana** [LedgerHQ/ledger-device-rust-sdk#216]
Doc at https://docs.rs/crate/ledger_device_sdk/latest meaning doc generate failed, it would be nice to fix it.

![image](https://github.com/user-attachments/assets/7702a2de-69e1-4f9b-adcd-7eb8098a4fcc)

**Comment by Pana** [comment]
Also the rust-analyzer not work when using docker image

**Comment by yogh333** [comment]
Yes that could be fine but it requires that a custom docker image is provided to doc.rs to build the doc. However this feature is still not supported 😞 : https://github.com/rust-lang/docs.rs/pull/1193
